csharp/test/Ice/properties/Client.cs

```diff
@@ -20,10 +20,10 @@ public override int
         run(String[] args)
         {
             Ice.Properties properties = communicator().getProperties();
-            test(properties.getProperty("Ice.Trace.Network").Equals("1"));
-            test(properties.getProperty("Ice.Trace.Protocol").Equals("1"));
-            test(properties.getProperty("Config.Path").Equals("./config/中国_client.config"));
-            test(properties.getProperty("Ice.ProgramName").Equals("PropertiesClient"));
+            test(properties.getProperty("Ice.Trace.Network") == "1");
+            test(properties.getProperty("Ice.Trace.Protocol") == "1");
+            test(properties.getProperty("Config.Path") == "./config/中国_client.config");
+            test(properties.getProperty("Ice.ProgramName") == "PropertiesClient");
             test(appName().Equals(properties.getProperty("Ice.ProgramName")));
```

Comment:
`appName()` returns a string too, doesn't it?

```
test(properties.getProperty("Ice.ProgramName") == appName());
```

Reply:
Yes, it was an automatic update. There are many more Equals that can be replaced by ==